tools/cd_scripts/install_test.sh

```diff
@@ -16,8 +16,16 @@
 # Print commands and their arguments as they are executed.
 set -x
 
+echo "Upgrade gcloud version"
```

Comment:
Do we have a way to re-use this?

I understand a single script is passed as a start-up script to VM, so would required more effort to re-use.

Reply:
> I understand a single script is passed as a start-up script to VM, so would required more effort to re-use.

Yes exactly. I thought so but we are adding one file in VM metadata and running that file only.